charts/drone/templates/_helpers.tpl

```diff
@@ -62,3 +62,16 @@ Create the name of the service account to use
     {{ default "default" .Values.serviceAccount.name }}
 {{- end -}}
 {{- end -}}
+
+{{/*
+TLS enabled via extraVolumes
+*/}}
+{{- define "drone.tlsEnabled" -}}
+  {{/* Iterate through each of the extraVolumes */}}
+  {{- range .Values.extraVolumes -}}
+    {{/* If a volume called certs exist TLS is enabled */}}
+    {{- if eq .name "certs" }}
+true
+    {{- end -}}
+  {{- end -}}
```

Comment:
There are two ways to allow drone to serve TLS: provide cert+key in a volume, or drone can use Let's Encrypt to manage a generated certificate.

What about inspecting `.Values.env` Instead of relying on the special volume name?
https://docs.drone.io/server/https/#certificate-upgrades

```suggestion
  {{/* Iterate through each of the extraVolumes */}}
  {{- if or (hasKey .Values.env "DRONE_TLS_AUTOCERT") (hasKey .Values.env "DRONE_TLS_CERT") -}}
true
  {{- end -}}
```

Reply:
Thanks Jacob, that's a much nicer solution.